src/apic-extension/azext_apic_extension/tests/latest/utils.py

```diff
@@ -380,3 +381,64 @@ def _get_apic_environment(self, **kwargs):
             template = 'To create an API Center Deployment an API Center Environment is required. Please add ' \
                        'decorator @{} in front of this preparer.'
             raise CliTestError(template.format(ApicEnvironmentPreparer.__name__))
+        
+class ApimServicePreparer(NoTrafficRecordingPreparer, SingleValueReplacer):
+    def __init__(self, name_prefix='clitest', length=24,
+                 parameter_name='apim_name', resource_group_parameter_name='resource_group', key='apim_name',
+                 enable_system_assigned_identity=False):
```

Comment:
Maybe call `enable_system_assigned_identity` as `use_system_assigned_identity` to make it more clear? We're not enable APIM's system assigned identity here.

Reply:
By the way, you may also remove this parameter and determine the actual behavior based on whether `USERASSIGNED_IDENTITY` exist? I saw you added code in each test case to determine whether to use user assigned identity or not. Since we will use user assigned identity for all identity if it exists. So it could make the code simplier if we check the user assigned identity existense here.